auth/src/main/java/com/firebase/ui/auth/util/data/PhoneNumberUtils.java

```diff
@@ -172,1187 +162,21 @@ private static String countryIsoForCountryCode(String countryCode) {
      * /google/i18n/phonenumbers/PhoneNumberUtil.java#L2395
      */
     private static String countryCodeForPhoneNumber(String normalizedPhoneNumber) {
-        final String phoneWithoutPlusPrefix = normalizedPhoneNumber
-                .replaceFirst("^\\+", "");
-        final int numberLength = phoneWithoutPlusPrefix.length();
+        String phoneWithoutPlusPrefix = normalizedPhoneNumber.replaceFirst("^\\+", "");
+        int numberLength = phoneWithoutPlusPrefix.length();
 
         for (int i = 1; i <= MAX_LENGTH_COUNTRY_CODE && i <= numberLength; i++) {
-            final String potentialCountryCode = phoneWithoutPlusPrefix.substring(0, i);
-            final Integer countryCodeKey = Integer.valueOf(potentialCountryCode);
+            String potentialCountryCode = phoneWithoutPlusPrefix.substring(0, i);
+            Integer countryCodeKey = Integer.valueOf(potentialCountryCode);
 
-            if (CountryCodeToRegionCodeMap.containsKey(countryCodeKey)) {
+            if (COUNTRY_TO_REGION_CODES.indexOfKey(countryCodeKey) >= 0) {
                 return potentialCountryCode;
             }
         }
 
         return DEFAULT_COUNTRY_CODE;
     }
 
-    private static Map<Integer, List<String>> createCountryCodeToRegionCodeMap() {
```

Comment:
These functions were kinda icky so I cleaned them up.

FYI: the cleanup is guaranteed to not include typos or missing entries since I discovered IntelliJ's structural search and replace which enables this kind of hardcore refactoring.